openlibrary/templates/home/stats.html

```diff
@@ -1,42 +1,46 @@
 $def with (stats=None)
 $if stats:
-    <div class="head">
+    <div id="home-stats">
+      <div class="head">
         <h2 class="inline">Around the Library</h2>
         <p class="inline sansserif large collapse">&nbsp;Here's what's happened over the last 28 days. More <a href="/recentchanges">recent changes</a>.</p>
-    </div>
+      </div>
 
-    <div class="chartHome">
-        <div class="chartShow" onClick="window.location.href='//archive.org/stats';" title="See all visitors to OpenLibrary.org">
+      <div id="home-stats-charts">
+        <div class="statschart">
+          <div class="chartShow" onClick="window.location.href='//archive.org/stats';" title="See all visitors to OpenLibrary.org">
```

Comment:
The above 3 lines use 3 different word break formats: hyphen, smash-together, and camel case. Definitely not a big deal, but consistency would be nice.

Also, inline `onClick` handlers are a little ick. (I know they're legacy.)